apps/experiments/tests/test_models.py

```diff
@@ -980,6 +981,29 @@ def test_get_version(self, experiment):
         assert experiment.get_version(1) == first_version
         assert first_version.get_version(1) == first_version
 
+    def test_archive_working_experiment_deletes_all_versions(self, experiment):
+        first_version = experiment.create_new_version()
+        second_version = experiment.create_new_version()
+        experiment.archive_working_experiment()
+        experiment.refresh_from_db()
+        first_version.refresh_from_db()
+        second_version.refresh_from_db()
+
+        assert experiment.is_archived is True
+        assert first_version.is_archived is True
+        assert second_version.is_archived is True
+
+    def test_archive_deletes_channels(self, experiment):
+        experiment_channel = ExperimentChannelFactory(experiment=experiment)
+        experiment_version = experiment.create_new_version()
+        experiment_version_channel = ExperimentChannelFactory(experiment=experiment_version)
```

Comment:
We wouldn't really have version channels, since all channels will be linked to the working experiment

Reply:
+1 - can remove this part of the test